components/molecules/BenefitCardHeaderSummary.js

```diff
@@ -0,0 +1,33 @@
+import propTypes from 'prop-types'
+import { SummaryTypes } from '../../objects/UniversalBenefit'
+
+export default function BenefitCardHeaderSummary(props) {
+  const t = props.locale
+  let typesWithLinks = [SummaryTypes.PaymentAmount, SummaryTypes.LatestStatus]
+
+  return (
+    <div>
+      <p>{t[props.summary.type].title}</p>
+      <p className="font-bold">
+        {props.summary.value}
+        <br />
+        {props.summary.status ?? null}
+      </p>
+      {!typesWithLinks.find((t) => t == props.summary.type) ? null : (
+        <p>
```

Comment:
Not sure if we need this surrounding <p> tag. Could be removed with some styling. If not, no problem!

Reply:
content should always be wrapped in a `p`, and links would be considered content. Buttons and headers would be the exclusion.